pac4j-saml-opensamlv5/pom.xml

```diff
@@ -173,6 +181,20 @@
             <version>${hazelcast.version}</version>
             <optional>true</optional>
         </dependency>
+        <dependency>
+            <groupId>org.mongodb</groupId>
+            <artifactId>mongodb-driver-sync</artifactId>
+            <optional>true</optional>
+        </dependency>
+        <dependency>
+            <groupId>org.mongodb</groupId>
+            <artifactId>bson</artifactId>
+            <optional>true</optional>
+        </dependency>
+        <dependency>
+            <groupId>org.mongodb</groupId>
+            <artifactId>mongodb-driver-core</artifactId>
```

Comment:
Why is this dependency not `optional`?

Reply:
I thought that this dependency should be explicitly added by the pac4j user.

Reply:
You're right. Should be fixed and optional now.